app/src/main/java/org/fossasia/openevent/app/events/EventListActivity.java

```diff
@@ -29,55 +30,70 @@
 import butterknife.ButterKnife;
 import timber.log.Timber;
 
-import static org.fossasia.openevent.app.utils.ViewUtils.showView;
-
-public class EventsActivity extends AppCompatActivity implements IEventsView {
-
-    @BindView(R.id.tvOrganiserName)
-    TextView organiserName;
-
-    @BindView(R.id.rvEventList)
+/**
+ * An activity representing a list of Events. This activity
+ * has different presentations for handset and tablet-size devices. On
+ * handsets, the activity presents a list of items, which when touched,
+ * lead to a {@link EventDetailActivity} representing
+ * item details. On tablets, the activity presents the list of items and
+ * item details side-by-side using two vertical panes.
+ */
+public class EventListActivity extends AppCompatActivity implements IEventsView, OnFragmentInteractionListener {
+
+    @BindView(R.id.event_list)
     RecyclerView recyclerView;
 
-    @BindView(R.id.progressBar)
-    ProgressBar progressBar;
+    @BindView(R.id.toolbar)
+    Toolbar toolbar;
 
-    public static List<Event> events = new ArrayList<>();
+    /**
+     * Whether or not the activity is in two-pane mode, i.e. running on a tablet
+     * device.
+     */
+    private boolean mTwoPane;
```

Comment:
Don't use hungarian notation of variables. Only Google in the whole world does and for all wrong reasons